src/extension.ts

```diff
@@ -44,6 +45,21 @@ export function activate(context: vscode.ExtensionContext) {
     createCreateQueryCommand()
   );
 
+  vscode.commands.registerCommand(
+    'faunadb.uploadGraphQLSchema',
+    uploadGraphqlSchemaCommand('merge', adminSecretKey, outputChannel)
```

Comment:
@BrunoQuaresma We will need to update these `uploadGraphqlSchemaCommand` calls the use the `adminSecretKey` only if we don't have a `FAUNA_KEY` value in our `.faunarc` file 👍